src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/media/index.js

```diff
@@ -2,6 +2,8 @@ import pick from 'ramda/src/pick';
 import path from 'ramda/src/path';
 import is from 'ramda/src/is';
 
+const FALLBACK_PLACEHOLDER_IMAGE_URL = `${process.env.SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN}${process.env.SIMORGH_PUBLIC_STATIC_ASSETS_PATH}images/media_placeholder.png`;
```

Comment:
Is it possible to have a branded image instead of the grey background image? in public/{service}/images/icons/icon-{size}.png e.g. https://news.files.bbci.co.uk/include/articles/public/pidgin/images/icons/icon-512x512.png

Reply:
It's just that there was mention of using the CPS poster image as the fallback, but I am not sure what that is, or where it comes from

Reply:
This would mean TC2 and CPS maps behave differently

From memory, the reason we went with the light gray image for the loading placeholder is that it's very subtle.

With the high-contrast brand images, there is an ugly flicker when the placeholder image gets replaced by the media player

Reply:
Talked about this at the 3 amigos just now - editorial were just misunderstanding the issue

They thought the image would display until the user clicked play - that was why they wanted the brand image.

What we're providing here is a loading indicator used exclusively on amp.  As soon as the iframe is loaded, the blank image will be replaced

Reply:
Does this image need/have alt text?

Reply:
The image is used as a placeholder within `<amp-iframe>` elements - it doesn't seem to need (or be possible to add) alt text - https://amp.dev/documentation/components/amp-iframe/#iframe-with-placeholder